### DISCUSSION TITLE
[Go] Booting from MicroSD & other questions

### DISCUSSION THREAD

**Discussion by GGORG0** [linux-surface/linux-surface#1280] [Q&A] [answered]
Hi, I'm considering buying a Surface Go 4 for school and pen note-taking.

I'm a long-time Linux-only user, but still wanna keep Windows for the first few weeks in case something doesn't work properly.

I know I can shrink the Windows partition and dual-boot that way, but I know that dual-booting on 1 drive sometimes causes issues, and you have to decide on partition sizes upfront.

So, can the Surface Go boot an OS from the microSD card?

That thing aside, is everything working properly on the Go 4? Is it worth buying it over the Go 3? How much is the performance boost?

Also, how long will the battery last with the screen on (medium brightness) and how can I squeeze as much lifetime out of it on Arch Linux?

Another question, will the pen (thick version) work fully? Buttons over Bluetooth, pressure sensitivity, battery level, palm rejection, eraser? Will I be able to customize what the buttons do, like on Windows?

Off topic, what are some good note-taking apps for Linux other than Xournal++?

Wow, that was long.
Thanks.

**Comment by qzed** [comment, marked as answer]
> So, can the Surface Go boot an OS from the microSD card?

The Go (or generally any Surface) can't directly boot from microSD. At least not the older versions and I think that hasn't changed with the newer ones. You can probably set it up so that the root and home partitions are on the SD card, but the boot partition would have to stay on the main SSD.

**Reply by qzed** [reply]
As note-taking app, maybe [rnote](https://github.com/flxzt/rnote) might work for you. But it seems a bit CPU hungry.
